noir/noir-repo/compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -625,6 +658,32 @@ impl Context {
         Ok(warnings)
     }
 
+    fn handle_ssa_call_outputs(
+        &mut self,
+        result_ids: &[ValueId],
+        output_values: Vec<AcirValue>,
+        dfg: &DataFlowGraph,
+    ) -> Result<(), RuntimeError> {
+        for result in result_ids.iter().zip(output_values) {
```

Comment:
Can you re-add `(result_id, output)` here to avoid the tuple element accesses later?

Reply:
done